src/components/Trade/hooks/useTradeAmounts.tsx

```diff
@@ -219,7 +219,10 @@ export const useTradeAmounts = () => {
           buyAssetUsdRate: usdRates.buyAssetUsdRate,
           sellAssetUsdRate: usdRates.sellAssetUsdRate,
           selectedCurrencyToUsdRate,
-          tradeFee: bnOrZero(formFees?.tradeFee).div(bnOrZero(usdRates.buyAssetUsdRate)),
+          buyAssetTradeFeeUsd: bnOrZero(formFees?.buyAssetTradeFeeUsd),
+          sellAssetTradeFeeUsd: bnOrZero(formFees?.sellAssetTradeFeeUsd).div(
```

Comment:
Something seems off here, why are we dividing a USD rate by a USD rate... TODO: investigate.